OJP/OJP_Trips.xsd

```diff
@@ -186,6 +239,32 @@
 			<xs:enumeration value="earliestArrival"/>
 			<xs:enumeration value="latestDeparture"/>
 			<xs:enumeration value="earliestArrivalAndLatestDeparture"/>
+			<xs:enumeration value="environmentalSafety">
+				<xs:annotation>
+					<xs:documentation>If set, favour "green" modes/lines such as bike sharing and (electric) trains, avoid or restrict modes/lines known for higher CO2 emissions such as (conventional) taxi, ridehailing or coach.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="extraSafe">
+				<xs:annotation>
+					<xs:documentation>High level of safety. If used, certain modes, lines or zones/districts known for lower safety, i.e. higher risk of accidents and crime, may be avoided, others may be preferred. This may depend on the actual, local or time of day situation.E.g. bike or scooter may be considered unsafe in some cities/districts while safe in others.</xs:documentation>
```

Comment:
```suggestion
					<xs:documentation>High level of safety. If used, certain modes, lines or zones/districts known for lower safety, i.e. higher risk of accidents and crime, may be avoided, others may be preferred. This may depend on the actual, local or time of day situation. E.g. bike or scooter may be considered unsafe in some cities/districts while safe in others.</xs:documentation>
```

Reply:
Is this extra safe (regarding crime) or things like more reliable lines or vehicles?